tutorials/using-edge/iotedge-tf-template-gpu.json

```diff
@@ -0,0 +1,157 @@
+{
+
```

Comment:
please remove all the empty lines

Reply:
Done. It was a cross-platform conversion side effect.